src/open_inwoner/scss/components/Spinner/Spinner.scss

```diff
@@ -0,0 +1,31 @@
+@keyframes spinner {
+  0% {
+    transform: translate3d(-50%, -50%, 0) rotate(0deg);
+  }
+  100% {
+    transform: translate3d(-50%, -50%, 0) rotate(360deg);
+  }
+}
+
+.loader-container {
+  &--hide {
+    display: none;
+  }
+
+  .rotate {
+    animation: rotate 2.5s linear infinite;
```

Comment:
I feel this is just a tiny bit too slow, even if you have to wait long - 1.5 seconds would feel better.
All else looking good!